scrapbook/models.py

```diff
@@ -418,9 +385,20 @@ def scraps_report(self, scrap_names=None, notebook_names=None, header=True):
             the scraps to display as reported outputs
         notebook_names : str or iterable[str] (optional)
             notebook names to use in filtering on scraps to report
+        include_data : bool (default: False)
+            indicator that data-only scraps should be reported
         header : bool (default: True)
             indicator for if the scraps should render with a header
         """
+
+        def trim_repr(data):
+            # Used to generate a smallish version of data for display purposes
```

Comment:
Convert to one line docstring `"""Generate a small data representation for display purposes"""`